### PR TITLE
[feat] Splashscreens for empty pages

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully implemented empty splashscreens for Products, Cart and Wishlist.

Features:
1. If applied product filters do not return any item, user is shown splashscreen
2.  If user has not added any item(s) to their wishlist or cart, user is shown splashscreen
3.  All splashscreens have meaningful descriptions and redirects where neccessary.

## Deployment
- For Wishlist and Cart, visit the respective pages from navbar
- For Products, set price filter to zero

https://deploy-preview-22--fueledup-store.netlify.app/

## Preview
image
image
image

### CODE REVIEW

File: src/pages/cart/CartItems.jsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import CartCard from "../../components/CartCard";
 import PriceSummary from "../../components/PriceSummary";
```
Comment (GuptaEesh) [inline src/pages/cart/CartItems.jsx]: Ideally you should prefer named exports over default exports. 
Why?:
- It helps you keep track of what exactly you have imported
- You can rename it using 'as' and you will know that I have renamed the imported thing.
- Default exports have a loop hole you can import them using any name making it difficult to keep track.

**Review by GuptaEesh — COMMENTED** [review]
Good use of grids.
Mentioned one comment other than that it looks good.
Good to merge

File: src/pages/cart/CartItems.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/pages/cart/CartItems.jsx]: Thanks for mentioning, yes I do use named exports now in my current projects.
This was old code hence why I have not updated it, but good you caught it
I have updated the code now, using named exports

**Review by Tanay-J — APPROVED** [review]
I liked the empty pages feature that this PR deals with. Will try implementing in my project as well.
Code is clean owing to good use of utility classes and conventional coding practices.
Approving the changes.

**Comment by Hamdrive** [comment]
Successfully reviewed by 2 team members, changes suggested have been implemented and deployment is working as expected.
Thus this resolves the PR and can be merged to dev and closing this PR

**Pull request merged**